build

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env php
 <?php
+use Phar;
```

Comment:
Would make sense to add some more PHP-CS-Fixer fixers, but it's a bit on @elfet side